pkg/controller/provider/container/ovirt/client.go

```diff
@@ -144,13 +144,13 @@ func (r *Client) auth() (user string) {
 
 //
 // Get system.
-func (r *Client) system() (s *System, err error) {
+func (r *Client) system() (s *System, status int, err error) {
```

Comment:
It seems that the return value _s_ should be called _system_. I don't think it is really used anywhere, but maybe it could just be fixed at the same time.